### PR TITLE
Chore/627 GitHub registry

### PR DESCRIPTION
**Pull request by gurjmatharu**

### CODE REVIEW

**Review by pbastia — CHANGES_REQUESTED** [review]
we're missing the changes to artifactory url in the `values.yaml` file of the helm chart.
@matthieu-foucault do you know if it's something we need to setup, or if we can just use a predetermined proxy?

**Comment by matthieu-foucault** [comment]
Looks like there is no remote repository setup for `ghcr.io`, so we'll have to stop using artifactory, which is okay since there are no cost/quotas.

See https://github.com/BCDevOps/developer-experience/blob/master/apps/artifactory/DEVHUB-README.md#how-do-i-find-out-what-repos-are-available to find out the available repos

**Review by matthieu-foucault — APPROVED** [review]
Looks good to me. A rebase might fix CI

**Review by pbastia — APPROVED** [review]

**Review by pbastia — CHANGES_REQUESTED** [review]
wait wait

File: chart/cas-cif/values.yaml [hunk 1]
```diff
@@ -6,9 +6,9 @@ replicaCount: 2
 
 image:
   app:
-    repository: artifacts.developer.gov.bc.ca/google-docker-remote/ggl-cas-storage/cas-cif-app
+    repository: ghcr.io/bcgov/cas-cif-schema/cas-cif-app
```
Comment (pbastia) [inline chart/cas-cif/values.yaml:9]: This should be:

```suggestion
    repository: ghcr.io/bcgov/cas-cif-app
```

**Review by pbastia — CHANGES_REQUESTED** [review]
something was copy pasted just a bit too quickly

**Review by pbastia — APPROVED** [review]

**Pull request merged**